### DISCUSSION TITLE
Development Roadmap - any plans for the future of Fooocus?

### DISCUSSION THREAD

**Discussion by mashb1t** [lllyasviel/Fooocus#2154] [Q&A] [answered]
@lllyasviel There are now 92 PRs open and many of them do fix significant bugs or implement helpful and requested features.
Is there any development roadmap which you're following, if yes, can you please share it here, and when can you or another reviewer handle the PRs?
Information about this matter would be highly appreciated.

**Comment by docppp** [comment]
i strongly agree. contributing without any knowledge if ones work will not be ignored is frustrating. and i dont mean that every pr should be merged in no time, but any feedback will be appreciated.

**Comment by lllyasviel** [comment, marked as answer]
We will come back soon at next model release.

I also enabled the merge permission with Collaborators now. Feel free to use it if absolutely necessary - many installation related things will broken one-click package user with embedded python since some of them do not have correct internet connection to PYPI.

**Reply by xhoxye** [reply]
Yes, any PR changes to requirements need to be carefully considered

**Reply by mashb1t** [reply]
Thank you very much for the feedback and your trust. I'll make sure to not take advantage of the newly granted privileges and will be merging PRs with upmost care.

**Reply by GuidoBartoli** [reply]
> We will come back soon at next model release.

Just out of curiosity, are you referring to Stable Cascade or Stable Diffusion 3?

**Reply by mashb1t** [reply]
StabilityAI has announced the availability of SD3 today, see their post on X: https://twitter.com/StabilityAI/status/1780599024707596508

> Today, we are pleased to announce the availability of Stable Diffusion 3 and Stable Diffusion 3 Turbo on the Stability AI Developer Platform API.
We have partnered with 
@FireworksAI_HQ, the fastest and most reliable API platform in the market, to deliver these models.
In keeping with our commitment to open generative AI, we aim to make the model weights available for self-hosting with a Stability AI Membership in the near future.
You can get started and learn more here: https://bit.ly/3W43FjY

=> The model should be available soon, can't wait!

**Reply by 3Diva** [reply]
@mashb1t I'm looking forward to is as well! I hope that Fooocus will be able to run the SD3 models.

**Comment by lgs777** [comment]
From the perspective of a user who can only make use of the technology, I am grateful for all your hard work
